src/utils/xrQSlim/MxQSlim.cpp

```diff
@@ -481,8 +480,7 @@ void MxEdgeQSlim::update_pre_contract(const MxPairContraction& conx)
         if (u == v1 || varray_find(star, u))
         {
             // This is a useless link --- kill it
-            bool found = varray_find(edge_links(u), e, &j);
-            VERIFY(found);
+            VERIFY(varray_find(edge_links(u), e, &j));
```

Comment:
`varray_find` is used to initialize `j` variable, and placing the function call into `VERIFY` breaks the code on Release configurations, because VERIFY macro expands to nothing in Release. So, `j` is being left uninitialized.
```suggestion
            [[maybe_unused]] bool found = varray_find(edge_links(u), e, &j);
            VERIFY(found);
```

Reply:
Oh very good catch